sdlf-utils/glue-jobs-deployer/glue-job.yaml

```diff
@@ -0,0 +1,95 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Engineering <<pipeline-dataset>> Glue Job"
+
+Parameters:
+    pPipelineBucket:
+      Description: "The artifactory bucket used by CodeBuild and CodePipeline"
+      Type: "AWS::SSM::Parameter::Value<String>"
+      Default: "/SDLF/S3/ArtifactsBucket"
+    pJobFiles:
+      Type: String
+      Description: The .sql and .json files from the jobs.
+    pPipeline:
+      Type: String
+      Description: The pipeline name (i.e. analyticspipeline).
+    pTeam:
+      Type: String
+      Description: The team name (i.e. engineering)
+    pDataset:
+      Type: String
+      Description: The dataset name (i.e. sampledataset)
+    pAllocatedCapacity:
+      Type: Number
+      Description: The glue job allocated capacity
+
+Resources:
+  rGlueRole:
+    Type: "AWS::IAM::Role"
+    Properties: 
+      RoleName: !Sub 'sdlf-${pPipeline}-${pDataset}-glue-role'
+      Path: "/service-role/"
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: "Allow"
+            Principal:
+              Service:
+                - "glue.amazonaws.com"
+            Action: 
+              - "sts:AssumeRole"
+      ManagedPolicyArns: 
+        - "arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"
```

Comment:
Perhaps we can add a comment here to say that customers should modify this list depending on what they require their Glue job to do?

Reply:
great idea, done, thank you!